src/bin/travis/testCentaurTes.sh

```diff
@@ -58,7 +58,7 @@ mkdir go-lib
 export GOPATH=$WORKDIR/go-lib
 go get github.com/ohsu-comp-bio/funnel
 cd $GOPATH/src/github.com/ohsu-comp-bio/funnel
-git checkout 04c5e03
+git checkout c4d9134
```

Comment:
This the head of develop for funnel currently. The reason is incorporating this fix https://github.com/ohsu-comp-bio/funnel/pull/164 that allows the call caching caopeira centaur test to survive a restart mid-test in TES.
However it seems to have broken another test, namely `non_root_default_user`. I excluded it for now but will follow up with the OHSU guys to figure out what's happening